azurerm/helpers/azure/network.go

```diff
@@ -0,0 +1,17 @@
+package azure
+
+import (
+	"fmt"
+	"net"
+)
+
```

Comment:
I don't think we need this function?

Reply:
That's correct, it was a leftover from earlier tries. 
Removed.